platform/ios/src/MGLMapView.mm

```diff
@@ -532,21 +532,21 @@ - (void)commonInit
     _singleTapGestureRecognizer.delegate = self;
     [self addGestureRecognizer:_singleTapGestureRecognizer];
 
-    _twoFingerTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(handleTwoFingerTapGesture:)];
-    _twoFingerTap.numberOfTouchesRequired = 2;
-    [_twoFingerTap requireGestureRecognizerToFail:_pinch];
-    [_twoFingerTap requireGestureRecognizerToFail:_rotate];
-    [self addGestureRecognizer:_twoFingerTap];
-
     _twoFingerDrag = [[UIPanGestureRecognizer alloc] initWithTarget:self action:@selector(handleTwoFingerDragGesture:)];
     _twoFingerDrag.minimumNumberOfTouches = 2;
     _twoFingerDrag.maximumNumberOfTouches = 2;
     _twoFingerDrag.delegate = self;
-    [_twoFingerDrag requireGestureRecognizerToFail:_twoFingerTap];
     [_twoFingerDrag requireGestureRecognizerToFail:_pan];
     [self addGestureRecognizer:_twoFingerDrag];
     _pitchEnabled = YES;
 
+    _twoFingerTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(handleTwoFingerTapGesture:)];
+    _twoFingerTap.numberOfTouchesRequired = 2;
+    [_twoFingerTap requireGestureRecognizerToFail:_pinch];
+    [_twoFingerTap requireGestureRecognizerToFail:_rotate];
+    [_twoFingerTap requireGestureRecognizerToFail:_twoFingerDrag];
```

Comment:
This PR does not change the average recognition delay for the two-finger tap gesture (0.056s before, 0.058s after — within the margin of error for my manual test, I’d say).

Note that I performed this test by timing from `touchesBegan:withEvent:` to `twoFingerTap.state == UIGestureRecognizerStateEnded`. I double-tapped from max zoom to min zoom, as fast as my fingers could tap the screen (2×) and also waiting for the zoom to finish (2×), then averaged the resulting ~200 measurements. Release build.